app/javascript/packs/utils/calendar.js

```diff
@@ -6,18 +6,18 @@ const Day = Immutable.Record({day: undefined, inner: undefined, today: false});
 const Week = Immutable.Record({days: Immutable.List()});
 
 export function prev(base){
-  return base.clone().subtract(1, 'M').date(base.date());
+  return base.clone().subtract(1, 'M');
 }
 
 export function next(base){
-  return base.clone().add(1, 'M').date(base.date());
+  return base.clone().add(1, 'M');
 }
 
 export function week(date, range){
```

Comment:
You modified this function but didn't add tests to guarantee it will always work as expected, should it have tests?

Reply:
I added this [test](https://github.com/Codeminer42/Punchclock/pull/168/files#diff-cbdcfd2dcd0b57fe9951a9ec03bdde02a407e0c94435d7c8af73d7a561d8c04dR37). What do you think about it?

Reply:
It makes sense to me 👍🏽